### ISSUE TITLE
Bug: Zoom with pinch gesture disabled for sign in modal on mobile and desktop

### ISSUE THREAD

**Issue by SarahKay8** [department-of-veterans-affairs/va.gov-team#58960]
## Issue Description

On the Sign-in modal i can't use ctrl+mouse wheel to zoom in on desktop (windows/chrome) and I can't use the pinch method to zoom in.

## How to reproduce 

1. logging into [staging ](staging.va.gov)(or prod), on a touch screen device. 
2. click "Sign-In" 
3. Try to zoom in on touch screen device or try to zoom in with mouse. No luck 

## Acceptance Criteria
- Enabled zoom gesture on Sign-In modal. 
---
## How to configure this issue
![73554997-2517-4613-8534-1C71260F7102_4_5005_c.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/c4c4b523-623a-42fb-bbfc-f5ac943448e9)- 
The viewport looks right however I am wondering if we can get rid of these two meta tags. `<meta name="HandheldFriendly" content="True">`
`<meta name="MobileOptimized" content="320">`
These meta tags aren't doing any harm but they aren't needed and they take up a few bytes of page loading and I'm not sure if this is contributing to the zoom issue. The `HandheldFriendly `meta tag was used for a very old version of BlackBerry.
REF: https://developer.blackberry.com/playbook/html5/documentation/handheldfriendly.html

The `MobileOptimized` meta tag was used for a very old version of Windows Mobile 5 / 6 (before even Windows Phone 7, 8 and Windows Mobile 10!).

There also seems to be event listeners in place somewhere (including listeners for `wheel, scroll, resize, keydown`, etc) on the page...i suspect one or more of them are possibly preventing this behavior. I'd like to brainstorm with DST on how we can remediate this.

**Comment by RLHecht** [comment]
@caw310 this ticket is for the design system team. Feel free to move it to your board if need be.

**Comment by coforma-terry** [comment]
Hey @sara-amanda - do we have an update on this? Thanks!

**Comment by sara-amanda** [comment]
> Hey @sara-amanda - do we have an update on this? Thanks!

@coforma-terry Looks like SK has Carol assigned to this ticket and [Randi mentions this should be DST](https://github.com/department-of-veterans-affairs/va.gov-team/issues/58960#issuecomment-1599097632), so perhaps now that Ryan is on the DST, this ticket might go to him? Thoughts?

**Comment by caw310** [comment]
@rsmithadhoc would you test to see if this is an issue and if it is DST problem?

**Comment by rsmithadhoc** [comment]
> [@rsmithadhoc](https://github.com/rsmithadhoc) would you test to see if this is an issue and if it is DST problem?

@caw310 I don't believe it is a DST issue. The sign in modal is not using the `va-modal` web component.

**Comment by caw310** [comment]
According to @rsmithadhoc this is not a DST issue. @SarahKay8 I'm not sure who this should be assigned to. Removing DST label.

**Comment by SarahKay8** [comment]
@caw310 I'm not quite sure who this goes to either since it's global? I'll check around
